.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 8.5.0
+current_version = 8.6.0
```

Comment:
As noted on [Slack](https://inmanta.slack.com/archives/CKRF0C8R3/p1688024923128089) it appears we've already been a bit too eager in bumping this.